troubleshooting/common_issues/index.rst

```diff
@@ -60,3 +60,20 @@ You can revert this operation by running the following commands:
     rm -rf ./app/cache/*
     php app/console pim:install:asset --env=prod
     php app/console assets:install --symlink web
+    
+I cannot export my products "An exception occurred during the export"
+--------------------------------------------------------------------
+
+If you have **thousands of products to export with the native csv or xslx connector** and if the job finishes or fails with the error: **"An exception occured during the export"**, first check the prod.log to have more information about the error.
+
+If the system mentions an error like "CRITICAL: Fatal Error: Allowed memory size of bytes exhausted (tried to allocate XXXXXXXX bytes)" **it means that there is a memory leak and it might be linked to the media archiving**. We have noticed that exporting more than 1 Gb of medias could lead to a memory leak :doc:`/reference/scalability_guide/more_than_1GB_of_product_media_to_export.html`
```

Comment:
And I think it would be a good idea to mention [uprofiler](https://github.com/FriendsOfPHP/uprofiler) or Blackfire if this is about dealing with custom developments. If it is native exports/imports, we won't ask them to use these tools, of course, .